app/backend/src/couchers/data.py

```diff
@@ -0,0 +1,40 @@
+import json
```

Comment:
I think "data" is a too anonymous name for a module. What about rodata or constant_tables?

Reply:
I've called it `constant_data` for now, do you think that's good enough?

We seem to keep on running into the second hard problem of CS, naming things